File: app/src/main/java/can/siempredelao/f1kotlin/backend/DriverStanding.java
package can.siempredelao.f1kotlin.backend;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class DriverStanding {

    public String            position;
    public String            positionText;
    public String            points;
    public String            wins;
    @SerializedName("Driver")
    public Driver            driver;
    @SerializedName("Constructors")
    public List<Constructor> constructors;

}

File: app/src/main/kotlin/can/siempredelao/f1kotlin/backend/model/DriverStanding.kt
package can.siempredelao.f1kotlin.backend.model

import com.google.gson.annotations.SerializedName

class DriverStanding(val position: String,
                     val positionText: String,
                     val points: String,
                     val wins: String,
                     @SerializedName("Driver") val driver: Driver,
                     @SerializedName("Constructors") val constructors: List<Constructor>)
